scripts/work.py

```diff
@@ -48,10 +48,25 @@ def convert(tile_index, time_bucket, tile_id):
 
     print('[INFO] Finished running conversion')
 
-def download_data(keys_array, s3_reporter_bucket, s3_datastore_bucket, time_key):
+def download_data(prefixes_array, s3_reporter_bucket, s3_datastore_bucket, time_key):
     client = boto3.client('s3')
     s3_resource = boto3.resource('s3')
-
+    
+    # get the keys
+    keys_array = []
+    for prefix in prefixes_array:
+        token = None
+        first = True
+        while first or token:
+            if token:
+                objects = client.list_objects_v2(Bucket=s3_reporter_bucket, Delimiter='/', Prefix=prefix)
+            else:
+                objects = client.list_objects_v2(Bucket=s3_reporter_bucket, Delimiter='/', Prefix=prefix, ContinuationToken=token)
+            keys_array.extend([ o['Key'] for o in objects['Contents']])
+            token = objects.get('NextContinuationToken')
+            first = False
```

Comment:
this is a bit annoying because of pagination, but basically we for each prefix, get all the objects. if there were more than the limit you can get in one go you have to get the continuation token and use that to get more (ie pagination)